browse/templates/base.html

```diff
@@ -92,13 +92,44 @@ <h1>arXiv.org</h1>
     {% block content %}
     {% endblock content %}
   </div>
-  <div id="footer">
-    <div class="footer-text">
-      <p>Link back to: <a href="{{url_for('home')}}">arXiv</a>, <a href="{{url_for('browse.form')}}">form interface</a>, <a href="{{url_for('contact')}}">contact</a>.&nbsp;&nbsp;<span class="help" style="display: inline-block; float: right"><a href="https://confluence.cornell.edu/x/MjmLFQ">Browse v0.1 released 2018-10-22</a>&nbsp;&nbsp;<button class="button is-small" id="feedback-button">Feedback?</button></span></p>
-      <p class="a11y-text" style="margin: 1em 0; padding-top: 1em; border-top: 1px solid #ccc">If you have a disability and are having trouble accessing information on this website or need materials in an alternate format, contact <a href="mailto:web-accessibility@cornell.edu">web-accessibility@cornell.edu</a> for assistance.</p>
+
+  {#- TODO: url_for on as many links as possible -#}
```

Comment:
arxiv-base has all of these links as url_for -- is it possible to copy config and code from there? Don't know how much trouble that would be.

Reply:
I'd like to upgrade to the latest version of arxiv-base in arxiv-browse while taking care of this TODO. I ran into some problems trying the upgrade before, so that's why I've pinned 0.12.1rc2 in `Pipfile` for now. I'll create a separate ticket/PR for the upgrade work.